corehq/apps/users/views/mobile/users.py

```diff
@@ -122,8 +122,10 @@ def custom_data(self):
     def editable_user(self):
         try:
             user = CommCareUser.get_by_user_id(self.editable_user_id, self.domain)
-            if not user or user.is_deleted():
+            if not user:
                 raise Http404()
+            if user.is_deleted():
+                self.template_name = "users/deleted_account.html"
```

Comment:
yeah, this came from a revert of a previous commit (as most of this stuff did). definitely magical, can i followup in another PR?